<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(139, 173, 227);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_gener">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0, 207, 100);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
       </property>
       <property name="text">
        <string>Генерация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb4">
       <property name="text">
        <string>4X4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb5">
       <property name="text">
        <string>5X5</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb6">
       <property name="text">
        <string>6X6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите режим</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_revers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
background-color: rgb(198, 0, 0);
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;

</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_restart">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 207, 207);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
padding: 1px;
</string>
    </property>
    <property name="text">
     <string>Рестарт</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_clear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>891</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
